src/main/java/it/eng/idsa/streamer/service/RejectionMessageService.java

```diff
@@ -25,7 +25,7 @@
 
 public class RejectionMessageService {
 
-    private static final String informationModelVersion = "2.1.0-SNAPSHOT"; //TODO Pom.xml
+    private static final String informationModelVersion = "4.1.1-SNAPSHOT"; //TODO Pom.xml
```

Comment:
We should change this value to 4.1.0, since this is major version supported by others.

Reply:
Removed field, now using value from UtilMessageService